test/integration_test/utils/marketplace.go

```diff
@@ -0,0 +1,251 @@
+package utils
+
+import (
+	"fmt"
+	"time"
+
+	v1 "github.com/operator-framework/api/pkg/operators/v1"
+	"github.com/operator-framework/api/pkg/operators/v1alpha1"
+	opmpops "github.com/portworx/sched-ops/k8s/operatormarketplace"
+	"github.com/portworx/sched-ops/task"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	defaultCatalogSourceName      = "test-catalog"
+	defaultCatalogSourceNamespace = "openshift-marketplace"
+	defaultOperatorGroupName      = "test-operator-group"
+	defaultSubscriptionName       = "test-portworx-certified"
+
+	getInstallPlanListTimeout       = 1 * time.Minute
+	getInstallPlanListRetryInterval = 2 * time.Second
+)
+
+// DeployAndValidatePxOperatorViaMarketplace deploys and validates PX Operator via Openshift Marketplace
+func DeployAndValidatePxOperatorViaMarketplace() error {
+	// Deploy CatalogSource
+	opRegistryImage := fmt.Sprintf("docker.io/portworx/px-operator-registry:%s", PxOperatorTag)
```

Comment:
nit: have a const for this catalog path.

Reply:
Done